src/components/AddTodoForm.js

```diff
@@ -1,40 +1,58 @@
-import React, { useState } from 'react'; 
-import PropTypes from 'prop-types';
-import InputWithLabel from './InputWithLabel';
+import React, { useState } from "react";
+import style from "./AddTodoForm.module.css";
+import PropTypes from "prop-types";
+import InputWithLabel from "./InputWithLabel";
 
 function AddTodoForm({ onAddTodo }) {
-  const [todoTitle, setTodoTitle] = useState('');
+  const [todoTitle, setTodoTitle] = useState("");
 
   const handleTitleChange = (event) => {
     const newTodoTitle = event.target.value;
     setTodoTitle(newTodoTitle);
   };
+
   function handleAddTodo(event) {
     event.preventDefault();
 
-    setTodoTitle('');
     onAddTodo(todoTitle);
+    setTodoTitle("");
   }
 
   return (
     <form onSubmit={handleAddTodo}>
-      <div className="input-container">
+      <div className={style.inputContainer}>
```

Comment:
there's something going on here where the `flex` div is extending way beyond the width of its parent: 


https://github.com/catbus00/react-todo/assets/12771651/3d0971e1-c14a-463b-bbdb-aade2add5a9b

I think this might also be preventing the items inside it from scaling properly

Reply:
I tried to debug it but this stuff can be really difficult, and I think this would be something we'd need to pair program on later

Reply:
although maybe it could be as simple as hiding the label or something at the smallest width

Reply:
I also added columns within the container to style this component in more detail, you can check with my commit message :)